test/integration.js

```diff
@@ -463,3 +477,69 @@ describe('core-node-express:app integration test as microservice', function () {
 
 });
 
+describe('core-node-express:app integration test as blank', function () {
+
+	// Express build is slow so we need to set a longer timeout for the test
+  this.timeout(150000);
+
+  before(function () {
+		//let swagger = JSON.parse(fs.readFileSync(path.join(__dirname, '../test/resources/person_dino.json'), 'utf8'));
+
+		// let swagStr = JSON.stringify(swagger);
+    return helpers.run(path.join(__dirname, '../app'))
+  .withOptions({
+    spec: JSON.stringify({ appname: 'testApp', port: common.defaultPort, applicationType: 'MS' }),
```

Comment:
is applicationType = MS the type we use for blank projects?

Reply:
No for Microservice lol